### PR TITLE
adding frontend functionality for the other wordlist indices

### PR DESCRIPTION
**Pull request by cmroughan**
Thanks for the updates to handle the different wordlist languages more cleanly! I noticed that the language switch wasn't handled in just one spot in the intro, so I'm inserting a quick fix for that.

### CODE REVIEW

**Comment by cmroughan** [comment]
Adding on another commit, this one to add the frontend functionality for the other wordlist indices.

**Comment by cmroughan** [comment]
And a quick addition to add intro text to the personal names list, as well.

File: src/routes/wordlist/[language]/+page.svelte [hunk 1]
```diff
@@ -50,7 +50,7 @@
 </h2>
 
 <p style="padding-left: 200px; padding-right: 80px;">
-	The following is the wordlist of Aramaic in the Inscriptions of Israel/Palestine. The list
+	The following is the wordlist of {initialCaps(currentLanguage)} in the Inscriptions of Israel/Palestine. The list
```
Comment (pletcher) [inline src/routes/wordlist/[language]/+page.svelte:53]: Thank you for catching this!

**Review by pletcher — APPROVED** [review]
I'm gong to merge this in. Sometime in the future we can circle back and clean up a few things/make more use of Svelte's reactivity, but I don't see any major issues.

**Pull request merged**